taxi/models.py

```diff
@@ -15,7 +15,12 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        null=True,
+        blank=True,
```

Comment:
The `license_number` field in the `Driver` model is set with `null=True` and `blank=True`, making it optional. If `license_number` is intended to be a required field, consider removing these options to enforce data integrity.